vanilla-framework/vanilla-framework
# Update docs styling to Vanilla 1.7.0-beta-2.1

## Done

- Updated docs styling to work with Vanilla 1.7.0 changes
- Split out the inline JS from template.html to static/js/scripts.js
- Created a settings file for Vanilla scss

## QA

- Pull code
- Run `cd docs && ./run`
- Open http://0.0.0.0:8104
- Check that there are no styling errors
- Note: `<pre>` blocks not wrapping, and `<code>` blocks having unnecessary top padding will be fixed in the 1.7.0 release - the `docs-1.7.0` branch will be updated when that happens.
